BUILD.bazel

```diff
@@ -233,6 +234,19 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "spirv_tools_lint",
+    srcs = glob(["source/lint/*.cpp"]),
```

Comment:
Don't we also need `.h` files in `srcs`? `source/opt` seems to put private headers together with `.cpp` files

Reply:
To be honest, I don't know, and I'm not very familiar with Bazel. It seems like all of the different tools are doing something different here, too. Can you ping someone who's more knowledgeable about this?

Reply:
The main difference between `srcs` and `hdrs` is that `srcs` are private while `hdrs` get exported to dependent targets: https://docs.bazel.build/versions/main/be/c-cpp.html#cc_binary_args

@dnovillo for bazel expertise: ^